apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -158,13 +158,23 @@ const IFrameSectionCodec = t.strict({
   src: t.string,
 });
 
+const FormCategoryCodec = t.strict({
+  key: t.string,
+  label: t.string,
+  additionalLabel: t.union([t.string, t.undefined]),
+});
+
+const FormCategoriesCodec = t.array(FormCategoryCodec);
```

Comment:
```suggestion
const FormCategoriesCodec = t.array(t.strict({
  key: t.string,
  label: t.string,
  additionalLabel: t.union([t.string, t.undefined]),
}));
```

If `FormCategoryCodec` isn't used elsewhere I'd just forgo it.